Docs: Update server-configuration.md to add link to SHOW PROPERTIES (#8592)
confluentinc/ksql: docs/reference/server-configuration.md

specified using the `ksql-server.properties` file. Some parameters, however,
can be set on a per-persistent query basis using `SET`. This is indicated in each parameter
section below.

> The current list of configuration settings can be retrieved using the [SHOW PROPERTY](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/show-properties/) command. 

Reviewer: ```suggestion
Retrieve the current list of configuration settings by using the [SHOW PROPERTY](/developer-guide/ksqldb-reference/show-properties/) command. 
```
